level_test.go

```diff
@@ -114,10 +114,11 @@ func TestListRubricLevels(t *testing.T) {
 	)
 	client := BestTestClient(t, "rubric/level/list", testRequest)
 	// Act
-	result, _ := client.ListLevels()
+	result, err := client.ListLevels(nil)
```

Comment:
This is the only customer facing change and since its a argument change in a type safe language customers code (or our other tools) will detect it at compile time